src/plotly.js

```diff
@@ -79,6 +79,12 @@ exports.register(require('./traces/scatter'));
 // plot api
 require('./plot_api/plot_api');
 exports.PlotSchema = require('./plot_api/plot_schema');
+// toImage to attach directly to Plotly
+//  to allow us to get rid of Snapshot below
+//  for version 2.0.0
+exports.toImage = require('./plot_api/to_image');
```

Comment:
If read correctly, then if I eliminate this line, `Plotly.toImage` will be undefined, since I do not attach `toImage` in [`plotly.js`](https://github.com/timelyportfolio/plotly.js/blob/master/src/plotly.js).  I'm happy to add this in `plotly.js`, but I wanted to confirm before doing.